etc/configure.py

```diff
@@ -12,9 +12,9 @@
 To use, create a configuration directory tree that contains any of these:
 
 * Requirements files named with this convention:
- - requirements_base.txt contains common requirements installed on all platforms.
- - requirements_win.txt, requirements_linux.txt, requirements_mac.txt and
-   requirements_posix.txt are os-specific requirements.
+ - requirements-py36_base.txt contains common requirements installed on all platforms.
```

Comment:
Why do we need a -py36 there? we will not have a configure script for each Python version

Reply:
Then what i use here ? is `requirements-(python-version)_base is fine?

Reply:
Dont worry it is comments . By the ./configure the detect the python version , it will find requirement files based on ur platform/os.